### PR TITLE
Oceans Landing: add curated curriculum resources to /oceans

### PR DESCRIPTION
**Pull request by Erin007**
Super quick tables to get [Mike's resources ](https://docs.google.com/document/d/1ymPmw6p0-mDvLV4T8A8r6l8UHLxBxw-Ydy0jolNvppI/edit) on to code.org/oceans 
Screen Shot 2019-11-26 at 11 22 45 AM
Screen Shot 2019-11-26 at 11 22 51 AM

Follow up: 
- [x] loc strings 
- [x] confirm audience

### CODE REVIEW

File: pegasus/sites.v3/code.org/views/ai_ml_resources_table.haml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    {
+      url: "http://cognimates.me/home/",
+      title: "Cognimates",
+      description: "An AI education platform for building games, programming robots and training.",
+      audience: "K-12 (reading required)"
+    },
+  ]
+
+%h2="Activities Powered by AI and ML"
+%table{style: 'border: 1; width: 970px'}
```
Comment (breville) [inline pegasus/sites.v3/code.org/views/ai_ml_resources_table.haml]: Will these hardcoded widths work with the responsive layout?

**Comment by Erin007** [comment]
Screen Shot 2019-11-26 at 11 49 49 AM
Removed audience column for AI Activities pre Mike's feedback.

File: pegasus/sites.v3/code.org/views/ai_ml_resources_table.haml (hunk 1 above)
Comment (Erin007) [reply pegasus/sites.v3/code.org/views/ai_ml_resources_table.haml]: oh good call. Updated to %. Table on small screen: 
Screen Shot 2019-11-26 at 11 54 06 AM

**Review by breville — APPROVED** [review]

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 2]
```diff
@@ -34,6 +34,37 @@
   hoc_overview_oceans_tutorial_header: 'AI For Oceans'
   hoc_overview_oceans_tutorial_desc: 'Learn about AI, machine learning, training data, and bias, while exploring ethical issues and how AI can be used to address world problems.'
 
+  activity: 'Activity'
+  name: 'Name'
+  description: 'Description'
+  audience: 'Audience'
```
Comment (islemaster) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml:40]: These are pretty generic - is there any chance they'll conflict with existing keys?

**Pull request merged**